internal/nodes/avi_model_evh_nodes.go

```diff
@@ -1059,7 +1062,18 @@ func (o *AviObjectGraph) BuildModelGraphForSecureEVH(routeIgrObj RouteIngressMod
 		evhNode.ServiceMetadata.Namespace = namespace
 		evhNode.ServiceMetadata.HostNames = hosts
 		if evhNode.SSLKeyCertAviRef != "" {
-			certsBuilt = true
+			//Logic to check host crd with sslcert field present for given host or not.
```

Comment:
Matching with dummy secret name (line no - 1037) would verify if the host is referred in a hostrule, so we may not need this check. 
cc @chauhanshubham